packages/repository/src/types/donation.types.ts

```diff
@@ -0,0 +1,20 @@
+import { Address, ChainId, Hex } from "@grants-stack-indexer/shared";
+
+export type Donation = {
+    id: string;
+    chainId: ChainId;
+    roundId: Address | string;
+    applicationId: string;
+    donorAddress: Address;
+    recipientAddress: Address;
+    projectId: string;
+    transactionHash: Hex;
+    blockNumber: bigint;
+    tokenAddress: Address;
+    amount: bigint;
+    amountInUsd: string;
+    amountInRoundMatchToken: bigint;
+    timestamp: Date;
+};
+
+export type NewDonation = Donation;
```

Comment:
why do we need this, i would rather keep just `Donation`

Reply:
i think is better and cleaner to have separate typing (or interfaces/classes) for the "select" class, the "insert" typing "update" typing, even if they exactly match